src/GraphQL/Language/AST/Variable.cs

```diff
@@ -28,5 +28,10 @@ public object Value
         /// Indicates if the variable value has been set.
         /// </summary>
         public bool ValueSpecified { get; private set; }
+
+        /// <summary>
+        /// Indicates if the variable's value is the variable default.
+        /// </summary>
+        public bool IsDefault { get; set; }
```

Comment:
I could combine these two properties into an `ArgumentSource` property called `Source`.  But what if it has no source?  Add `None` enumeration member?

Reply:
not worth it